apps/web/src/app/pages/trade/supply/_components/LiquidityDetails/PoolRecord.tsx

```diff
@@ -0,0 +1,210 @@
+
+import PoolLogoWithNetwork from '@/app/components/PoolLogoWithNetwork';
+import RewardsDisplay from '@/app/components/RewardsDisplay/RewardsDisplay';
+import Skeleton from '@/app/components/Skeleton';
+import { useRatesWithOracle } from '@/queries/reward';
+import { formatBigNumber } from '@/utils';
+import { formatSymbol, getFormattedNumber } from '@/utils/formatter';
+import { CurrencyAmount, Token } from '@balancednetwork/sdk-core';
+import { getFormattedExternalRewards, getFormattedRewards, stakedFraction, totalSupply } from '../utils';
+import { getExternalShareReward, getShareReward } from './WithdrawPanel';
+
+const DashGrid = styled.div`
+  display: grid;
+  grid-template-columns: 4fr 5fr;
+  gap: 10px;
+  grid-template-areas: 'name supply action';
+  align-items: center;
+
+  & > * {
+    justify-content: flex-end;
+    text-align: right;
+
+    &:first-child {
+      justify-content: flex-start;
+      text-align: left;
+    }
+  }
+
+  ${({ theme }) => theme.mediaWidth.upSmall`
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-areas: 'name supply share rewards action';
+  `}
+`;
+
+export const HeaderText = styled(Typography)`
+  font-size: 14px;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  display: flex;
+  gap: 5px;
+  align-items: center;
+`;
+
+const DataText = styled(Flex)`
+  font-size: 16px;
+  justify-content: center;
+  align-items: end;
+  flex-direction: column;
+`;
+
+const StyledArrowDownIcon = styled(ArrowDownIcon)`
+  width: 10px;
+  margin-left: 10px;
+  transition: transform 0.3s ease;
+`;
+
+const StyledDataText = styled(Flex)`
+  font-weight: bold;
+`;
+
+const ListItem = styled(DashGrid)`
+  padding: 20px 0;
+  color: #ffffff;
+`;
+
+const APYItem = styled(Flex)`
+  align-items: flex-end;
+  line-height: 25px;
+`;
+
+export const PoolRecord = ({
+  poolId,
+  pool,
+  pair,
+  pairData,
+  balnReward,
+  boostData,
+  externalRewards,
+}: {
+  pool: Pool;
+  pair: Pair;
+  pairData?: PairData;
+  poolId: number;
+  balnReward: BigNumber;
+  externalRewards: CurrencyAmount<Token>[] | undefined;
+  boostData: { [key in string]: Source } | undefined;
+  apy: number | null;
+}) => {
+  const { xChainId } = pool;
+  const [baseAmount, quoteAmount] = usePoolTokenAmounts(pool);
+
+  const upSmall = useMedia('(min-width: 800px)');
+  const prices = useRatesWithOracle();
+  const stakedLPPercent = useStakedLPPercent(poolId); // TODO
+  const pairName = `${formatSymbol(baseAmount.currency.symbol) || '...'}/${formatSymbol(quoteAmount.currency.symbol) || '...'}`;
+  const sourceName = pairName === 'sICX/BTCB' ? 'BTCB/sICX' : pairName;
+
+  const { baseValue: baseWithdrawValue, quoteValue: quoteWithdrawValue } = useWithdrawnPercent(poolId) || {};
+  const balances = useBalance(poolId);
+  const stakedFractionValue = stakedFraction(stakedLPPercent);
+  const totalbBaln = useTotalSupply();
+  const userBbaln = useBBalnAmount();
+  const reward = getShareReward(
```

Comment:
@swiftcc624 It's not an issue of small reward amounts, the issue is not displaying rewards for spoke chain LPs at all. You have to provide LP balance to this function and can ignore bBALN and boosting information in case of spoke LP, there will be no boosts for spoke LPs until bBALN is cross-chain, as well.